bin/node/src/commands/store.rs

```diff
@@ -163,18 +162,13 @@ impl StoreCommand {
                 })?;
         }
 
-        // Write the genesis state to disk. This is used to seed the database's genesis block.
+        // Bootstrap the store database.
         let accounts = accounts.into_iter().map(|account| account.account).collect();
         let genesis_state = GenesisState::new(accounts, version, timestamp);
-        let genesis_output = data_directory.join(miden_node_store::GENESIS_STATE_FILENAME);
-        File::create_new(&genesis_output)
-            .and_then(|mut file| file.write_all(&genesis_state.to_bytes()))
-            .with_context(|| {
-                format!("failed to write genesis data to file {}", genesis_output.display())
-            })
+        Store::bootstrap(genesis_state, data_directory)
     }
 
-    fn generate_account(input: AccountInput, rng: &mut ChaChaRng) -> anyhow::Result<AccountFile> {
+    fn generate_account(input: &AccountInput, rng: &mut ChaChaRng) -> anyhow::Result<AccountFile> {
```

Comment:
There is a `TODO` comment on line 189 below. We can replace it with a brief explanation. Basically, we need to set nonce to a non-zero value because `ZERO` nonce means that an account is not yet on-chain. For example, when we first create an account locally, it gets nonce `ZERO`. Once we execute a transaction against it (which is required to record the account on-chain), the nonce will be incremented. So, no account recorded in a block has nonce `ZERO`.

Accounts created in the genesis block are different since they don't require transactions executed against them to get recorded. But we still want to uphold the invariant of non-zero nonce because in several places we treat going from `ZERO` to non-zero nonce differently than incrementing a non-zero nonce.

Reply:
Ah. I see, thanks. I added that `TODO` in the last PR and forgot to follow-up in the review :)